src/client.ts

```diff
@@ -84,16 +87,11 @@ export class Client {
    * Creates an Axios client with default options and logging attached.
    * @internal
    */
-  private createAxios(config?: AxiosRequestConfig, disableLogging?: boolean): AxiosInstance {
+  private createAxios(config?: AxiosRequestConfig): AxiosInstance {
     const instance = axios.create(config);
     instance.defaults.headers = this.options.extraHeaders || {};
-    if (disableLogging) {
```

Comment:
Without this condition, the request and response body of the OAuth 2.0 credentials exchange are logged. That’s a security issue we must defend against. (https://cwe.mitre.org/data/definitions/532.html)

Reply:
Good catch, intended change in 9848f05c0249e32457efcfa0cafc23f98a124dc0